src/brain/apis/getOwnershipData.ts

```diff
@@ -1,12 +1,12 @@
-import { GETSENTRY_ORG, SENTRY_REPO_SLUG } from '@/config';
+import { GETSENTRY_ORG } from '@/config';
 
-const OWNERSHIP_FILE_PATH =
-  'src/sentry/apidocs/api_ownership_stats_dont_modify.json';
+const OWNERSHIP_FILE_PATH = 'api_ownership_stats_dont_modify.json';
+const SENTRY_API_SCHEMA = 'sentry-api-schema'
```

Comment:
```suggestion
const SENTRY_API_SCHEMA = 'sentry-api-schema';
```

Another option is to inline this. The only reason we have the SENTRY_REPO_SLUG constant is to make testing easier, but in this case, this is a constant and can be inlined in the getContent call.